examples/dave_demo_launch/launch/dave_mud_demo.launch

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
```

Comment:
This is probably another good thing to add to the meeting agenda to discuss in a broader context - a lot of our launch files are very similar, and it would be possible to pull the common elements into one file.  On the other hand, having the major components explicitly listed in each launch file makes them more stand-alone and easier for community users to borrow and modify.
For right now, I would rather not include dave_demo.launch until/ unless we have that discussion.  In addition to the extra grab bar, there's also the spurious call to publish the sand_heightmap that throws the error you mentioned above.